qa/cephfs/begin/0-install.yaml

```diff
@@ -50,6 +58,14 @@ tasks:
         - xfsdump
         - xfsprogs
         - xfsprogs-devel
+        - inih-devel
+        - userspace-rcu-devel
+        - libblkid-devel
+        - gettext
+        - libedit-devel
+        - libattr-devel
+        - device-mapper-devel
+        - libicu-devel
```

Comment:
The RHEL couldn't find and install the `userspace-rcu-devel`, `libedit-devel` and `device-mapper-devel` packages and will fail.

I will move this deps installing to `xfstests_dev.py` and `fscrypt.sh` and will only support the `centos`, `centos-stream` and `ubuntu` distros for now.

Will update it soon.